geomstats/_backend/pytorch/__init__.py

```diff
@@ -699,6 +700,12 @@ def cumsum(x, axis=None):
     return torch.cumsum(x, dim=axis)
 
 
+def cumprod(x, axis=None):
+    if axis is None:
```

Comment:
This code is not tested. Could you add a unit test in `test_backends.py`? Thanks!